servicetalk-dependencies/build.gradle

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright © 2018-2019 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-core"
+apply plugin: "java-platform"
+
+description = "ServiceTalk BOM that includes direct dependencies"
+
+javaPlatform {
+  allowDependencies()
+}
+
+rootProject.subprojects.findAll {
```

Comment:
i think this lists all servicetalk modules like the `servicetalk-bom`.  if so, this can potentially be something like `api platform(project("servicetalk-bom"))`

essentially, in maven pom, it can translate to the bom import of the `servicetalk-bom`

Reply:
I had meant to try that, but forgot before finishing up this PR. I have now tested it and it does work so I am now including `servicetalk-bom` rather than repeating what that bom does.